framework/widgets/ActiveForm.php

```
@@ -210,12 +210,11 @@ public function init()
-        if (
-            Yii::$app?->useJquery &&
-            $this->enableClientScript &&
-            !$this->clientScript instanceof ActiveFormJqueryClientScript
-        ) {
-            $this->clientScript ??= ['class' => ActiveFormJqueryClientScript::class];
+        if ($this->clientScript === null && (Yii::$app->useJquery ?? false) && $this->enableClientScript) {
+            $this->clientScript = ['class' => ActiveFormJqueryClientScript::class];
+        }
+
+        if ($this->clientScript !== null && !$this->clientScript instanceof ClientScriptInterface) {
             $this->clientScript = Yii::createObject($this->clientScript);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Yii' in method 'init'. (undefined)

(StaticAccess)